### ISSUE TITLE
[CAIA Intake] Public Websites: Discharge Upgrade Wizard

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-team#74787]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Public Websites
- **OCTO product owner:** Dave Conlon
- **Product name:** Discharge Upgrade Wizard
- **Product manager:**  Fran Cross
- **Slack channel:** #[sitewide-public-websites](https://dsva.slack.com/archives/C52CL1PKQ) 
- **Dedicated content writer on your team (if you have one):** Jordan Woods
- **Dedicated a11y specialist on your team (if you have one):** Laura Flannery
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [x] Something else:  IA: temporary route to use for rebuild of an existing app that can be used in Staging

### What's the nature of your initiative and desired outcomes?

The existing discharge upgrade wizard was built in Vets.gov days. It does not use current design or content patterns. It requires a lot of scrolling due to the legacy pattern. We are also unsure if the information within the wizard remains correct.

This effort will improve the ability for Veterans to determine their eligibility for a discharge upgrade by re-modernizing the product to align with current VA.gov design and content standards, and convert to a subtask pattern.

To this end, we are requesting a URL so that we can test it in Staging prior to cutting over to the new code in Production. Please see this Slack thread for information from the engineers: https://dsva.slack.com/archives/C52CL1PKQ/p1705605019314219

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [X] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

- We are ready to start implementation.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [X] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** N/A

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Supporting artifact - Discharge Upgrade Wizard Initiative](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-military-records/discharge-upgrade-wizard/initiatives/2024-discharge-upgrade-wizard-subtask/initiative-brief.md)

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia Slack thread](https://dsva.slack.com/archives/C01K37HRUAH/p1706568539851279) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [x] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [x] Initial Review - @coforma-terry and @strelichl
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
```

**Comment by kristinoletmuskat** [comment]
Hey! Just popping in to say I got your ticket and I am going to talk to Mikki about this on Thursday.

**Comment by kristinoletmuskat** [comment]
Update: We discussed this in more detail in this [slack thread](https://dsva.slack.com/archives/C52CL1PKQ/p1705605019314219).

We decided that the team does not need a staging URL, but we will provide sample URLs for the questions of the flow in line with guidance provided to other teams that are updating the wizard pattern to the subtask pattern.

**Comment by kristinoletmuskat** [comment]
Hey hey this all looks pretty straightforward -- I see in your mural documentation that there might be a few more screens than show in the sketch file, which is fine. I think our main guidance is to not make unnecessary subdirectories, and just nest all of the questions under the base URL. So if you need to tweak any of these or add, feel free.

So, base URL will stay the same as: `va.gov/discharge-upgrade-instructions/`

And these child pages would sit under that:

/branch-of-service
/discharge-year
/reason
/changes-requested
/court-martial
/previously-applied
/previously-applied-date
/earlier-honorable-service
/review
/results


lmk if you have any Qs!

**Comment by FranECross** [comment]
@kristinoletmuskat Thanks so much for the quick turnaround and information! 🎉 I'll defer to @chriskim2311 for any technical questions upon his return. cc @jilladams

**Comment by chriskim2311** [comment]
@kristinoletmuskat @FranECross Following up from the slack thread and the messages here. I think this is a good solution that I can work with. Thanks for your help!

**Comment by FranECross** [comment]
Request has been fulfilled; closing ticket.

**Comment by jilladams** [comment]
Thread about next steps for review: https://dsva.slack.com/archives/C01K37HRUAH/p1725997353585629

**Comment by aprocik1** [comment]
Fran confirmed that we can close this intake ticket as complete. I'm moving it to validate for now.

**Comment by aprocik1** [comment]
The product team is launching phase 1 updates to the discharge upgrade wizard on Monday 12/16. Fran confirmed that we can close this ticket as complete.
